setup.py

```diff
@@ -16,6 +16,7 @@
 setup(
     name='spotify_dl',
     version=version,
+    python_requires='>=3',
```

Comment:
This only tells pip>=9.0 that the package is for python3 only. For older pip versions, it'll still get installed and fail. Do you want the package to support python2?